GLTFSDK.Test/Source/MeshPrimitiveUtilsTests.cpp

```diff
@@ -1323,6 +1323,481 @@ namespace Microsoft
                     Assert::AreEqual(outputIndices[262138], 131069U);
                     Assert::AreEqual(outputIndices[262139], 0U);
                 }
+
+                GLTFSDK_TEST_METHOD(MeshPrimitiveUtilsTests, MeshPrimitiveUtils_Test_SerializeTriangulatedIndices16_Triangles)
```

Comment:
You also need to add tests which start with a fan/strip/etc, and then do a round-trip by calling both the triangulate and reverse operations and then compare input to output.

Reply:
Added